front/app/utils/FormattedCurrency/getFormattedBudget.ts

```diff
@@ -0,0 +1,16 @@
+import { Locale } from 'typings';
```

Comment:
Kept this function in case we need a string instead of a JSX element (FormattedCurrency, see below)